static/scss/collection.scss

```diff
@@ -233,25 +246,33 @@ $video-card-width: 19%;
         }
 
         img {
-          width: 100%;
+          height: 100%;
+          position: absolute;
+          margin: auto;
+          top: 0;
+          left: 0;
+          bottom: 0;
+          right: 0;
```

Comment:
I think we can just set `top` and `left` here. I'm also not sure if `margin: auto;` is doing anything

Reply:
When I am testing this and I remove these things it seems to break the layout. I am testing this with thumbnails of different aspect ratios. Some are widescreen (16:9), some are standard (4:3), others are vertical, and some are extremely wide. If I remove the margin, and left and right attributes, the 4:3 videos are no longer centered. Did you test this with different aspect ratios?